jobs/ci-kubernetes-e2e-kubeadm-gce.sh

```diff
@@ -26,9 +26,18 @@ readonly testinfra="$(dirname "${0}")/.."
 
 export PROJECT="k8s-jkns-e2e-kubeadm-gce-ci"
 export KUBERNETES_PROVIDER=kubernetes-anywhere
+
+# This job only runs against the kubernetes repo, and bootstrap.py leaves the
+# current working directory at the repository root. Grab the SCM_REVISION so we
+# can use the .debs built during the bazel-build job that should have already
+# succeeded.
+export SCM_VERSION=$(./hack/print-workspace-status.sh | grep ^STABLE_BUILD_SCM_REVISION | cut -d' ' -f2)
```

Comment:
@fejta That may be the case when the job runs under Jenkins and uses `dockerized-e2e-runner.sh`, but this is a prow job, so it has its own custom Docker image. Prow launches the image defined in `config.yaml`, and the rest of the chain is the above, with the job calling `e2e-runner.sh` instead of `dockerized-e2e-runner.sh` to avoid Docker-in-Docker.

@krzyzacy Sounds good to me. Did you have any other feedback about the diff, or was that a very informal LGTM? :-)